<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LittleMarketHelper</class>
 <widget class="QMainWindow" name="LittleMarketHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LittleMarketHelper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Initialize" name="InitializeBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Edit" name="EditBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Calibrate" name="CalibrateBtn">
        <property name="minimumSize">
         <size>
          <width>129</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="History" name="HistoryBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Help" name="HelpBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="About" name="AboutBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Exit" name="ExitBtn">
        <property name="minimumSize">
         <size>
          <width>129</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="PortfolioValueLabel" name="PortfolioValueLbl">
        <property name="text">
         <string>Portfolio Value: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="PortfolioTableWidget" name="PortfolioTbl"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PortfolioTableWidget</class>
   <extends>QTableWidget</extends>
   <header>portfoliotablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>Initialize</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>History</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>About</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Help</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Calibrate</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Edit</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Exit</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PortfolioValueLabel</class>
   <extends>QLabel</extends>
   <header>portfoliovaluelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
